presto-client/src/main/java/com/facebook/presto/client/OkHttpUtil.java

```diff
@@ -170,6 +182,31 @@ public static void setupSsl(
         }
     }
 
+    private static KeyStore loadTrustStore(File trustStorePath, Optional<String> trustStorePassword)
+            throws KeyStoreException, IOException, NoSuchAlgorithmException, CertificateException
+    {
+        KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
+
+        // attempt to read the trust store as a PEM file
+        try {
+            List<X509Certificate> certificateChain = PemReader.readCertificateChain(trustStorePath);
+            if (!certificateChain.isEmpty()) {
+                trustStore.load(null, null);
+                for (X509Certificate certificate : certificateChain) {
+                    X500Principal principal = certificate.getSubjectX500Principal();
+                    trustStore.setCertificateEntry(principal.getName("RFC2253"), certificate);
```

Comment:
You can use `getName()` which is the same. BTW, there's also a `X500Principal.RFC2253` constant